app/navigation/Navigator.scala

```diff
@@ -224,10 +222,14 @@ class Navigator @Inject() () {
     case RaRReplacementsOfDomesticGoodsPage =>
       taxYear => _ => _ => RaROtherCapitalAllowancesController.onPageLoad(taxYear, NormalMode)
     case RaROtherCapitalAllowancesPage =>
+      taxYear => _ => _ => RaRAllowancesCheckYourAnswersController.onPageLoad(taxYear)
+    case JointlyLetPage(RentARoom) =>
+      taxYear => _ => _ => TotalIncomeAmountController.onPageLoad(taxYear, NormalMode, RentARoom)
+    case ClaimExpensesOrRRRPage(RentARoom) =>
       taxYear =>
         _ =>
           _ =>
-            RaRAllowancesCheckYourAnswersController.onPageLoad(taxYear)
+            controllers.ukrentaroom.routes.CheckYourAnswersController.onPageLoad(taxYear)
```

Comment:
Shouldnt this be RaRAllowancesCheckYourAnswersController?

Reply:
@modivilioglu this looks right to me. If I redirect this page to _**RaRAllowancesCheckYourAnswersController**_ it will to the allowances CYA page for rent a room. I want this to go to CYA page in the About section.